app/src/main/java/com/keylesspalace/tusky/fragment/AccountMediaFragment.kt

```diff
@@ -133,10 +133,12 @@ class AccountMediaFragment : BaseFragment() {
         val columnCount = context?.resources?.getInteger(R.integer.profile_media_column_count) ?: 2
         val layoutManager = GridLayoutManager(context, columnCount)
 
-        val lightThemeEnabled = PreferenceManager.getDefaultSharedPreferences(context)
-                .getBoolean("lightTheme", false)
-        val bgRes = if (lightThemeEnabled) R.color.window_background_light
-        else R.color.window_background_dark
+        val windowBackgroundTv = TypedValue()
+        activity?.theme?.resolveAttribute(
+                R.attr.window_background, windowBackgroundTv, true)
+
+        val bgRes = windowBackgroundTv.resourceId
+
```

Comment:
there should already be a helper method for this in com.keylesspalace.tusky.util.ThemeUtils (if not, add one)